### ISSUE TITLE
Steps for installing Bazelisk are not new-developer friendly

### ISSUE THREAD

**Issue by armandomontanez** [bazelbuild/bazelisk#571]
The [current instructions for installing Bazelisk](https://github.com/bazelbuild/bazelisk/tree/24935f4ce663d6eb79e428005b763c059b508866?tab=readme-ov-file#installation) are not user-friendly.

Generally, the widely expected ways to get a tool on the following platforms are:

- Windows: Download an `.exe` installer that guides users through installation.
- macOS: Download .deb or .pkg that guides users through installation.
- Linux: `apt get install bazelisk` or similar through a preferred package manager.

Today, the instructions are:

- Windows: `choco install bazelisk`
- macOS: `brew install bazelisk`
- Linux: There's essentially a decision tree here.

Often, when I direct people to install Bazel via https://bazel.build/install, they end up directly using a release of Bazel rather than Bazelisk. I know the reason I've done this in the past is because of the lack of clarity in the installation process for Bazelisk.

- On macOS, I don't use brew (I've had it corrupt frequently for various reasons, so I now avoid it entirely).
- On Windows, I've never once used or installed choco despite doing a decent amount of development on Windows.
- On Linux, I know how to take a binary and add it to `PATH`, but I know for a fact that someone just getting started with development would stumble over that.

Generally speaking, the recommended flows for installing Bazelisk are atypical at best, and frustrating at worst. Bazelisk should strive to make the installation experience as frictionless as possible to ensure Bazel is approachable for new developers.

**Comment by jwnimmer-tri** [comment]
As a step in the right direction, I think attaching Debian binary packages to the https://github.com/bazelbuild/bazelisk/releases page would be a nice start.  I opened a PR for that at https://github.com/bazelbuild/bazelisk/pull/563.

**Comment by eabase** [comment]
I totally agree. Trying to use this for the very first time I find the instructions lacking. 
There is no way I'm gonna install `choco` when I can just run the stand-alone release installer or use `winget`. 

At the end of the day, I still fail to see why this is needed. Long text there, but not very to the point. What is the problem Bazel, that this is trying to solve? (A use case scenario would have been enlightening.)

Then also for building, you are not saying anything on the requirements of having WiX and the unknown `Make-MSI` package, used in here:
* https://github.com/bazelbuild/bazel/blob/master/scripts/packages/msi/make_msi.ps1

**Comment by SpartanJoe193** [comment]
Absolutely. I've gotten interest in a program called QuarkGL, which I have been trying to compile Bazelisk for

**Comment by bingjeff** [comment]
+1. I actually stumbled onto this thread, because I wasn't sure this was "the right way", surely I was missing something? Nope.
